src/Database/DatabaseConfig.php

```diff
@@ -158,4 +201,30 @@ public function manager(): Contracts\TenantDatabaseManager
 
         return $databaseManager;
     }
+
+    /** Get the TenantDatabaseManager for this tenant's connection. */
+    public function hostManager(): Contracts\TenantDatabaseManager
+    {
+        // DB cache the previous connection configurations and changing the connection configuration does not work
+        // So we purge the previous host connection before changing it
+        $this->purgeHostConnection(); // todo come up with a better name
+
+        // Set the host connection configuration array fetched from the tenant's config and manipulated to form a connection
```

Comment:
What are you trying to say here? The comment is confusing. Either rewrite it or share what the point is meant to be and I'll write it for you.

Reply:
I'm trying to explain that we get the modified configuration array and set the host connection configuration. You can write a better one, but I don't think we need comments here. The code is straightforward. We are pushing the `hostConnection` config in the `connections` array. `hostConnection` does require some good comments because that's the method doing all of the modifications in the config. And we have one review related to this method.

Reply:
I've removed the comments related to `hostConnection` method. It was creating confusion. `hostConnection` method itself has comments so I think we don't need one here.